src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 3 && days < 7) {
+    return (days * 40) - 20;
+  }
+
+  if (days >= 7) {
+    return (days * 40) - 50;
+  }
+
+  return days * 40;
```

Comment:
days * 40 is repeating 3 times, move it into a variable and reuse it throughout your code